README.md

```diff
@@ -1,4 +1,29 @@
 # Why Journal
+
+## Setup
+1. Clone the repo.
+2. Install node packages. We are using node version 12.11.1
+```
+npm install
+```
+3. Set up database and test db
```

Comment:
We might need to let people know that they'll need postgreSQL, we should provide a link to setup instructions here